stefaniaPopescu26/assignment2/AncestorsTest.java

```diff
@@ -0,0 +1,54 @@
+import org.junit.Test;
+
+import java.util.Vector;
+
+import static org.junit.Assert.*;
+
+public class AncestorsTest {
+
+    public Ancestors<Integer> getTree() {
```

Comment:
As per comment in the tests for the Tree class: can you try to name the test cases in a way that would reflect what is being tested? E.g. testGetAncestors_invalidNode. testGetAncestors_rootNode, etc.